### ISSUE TITLE
Issue when pushing changes to master

### ISSUE THREAD

**Issue by NickeManarin** [Rolanddoda/vue-cli-plugin-gh-pages-auto-deploy#1]
Hi,

My push is failing for this reason:

`! [remote rejected] master -> master   
(refusing to allow an OAuth App to create or update workflow '.github/workflows/gh-pages-deploy.yml' without 'workflow' scope)`

Is this something that I can change/fix locally?

**Comment by NickeManarin** [comment]
I manually created the *.yml file, so at least the publishing is working now.

**Comment by Rolanddoda** [comment]
@NickeManarin did you just installed the plugin and pushed to master ? If you did something else, can you please share the steps to reproduce this issue ?

Btw, a similar issue is referenced [here](https://github.community/t5/GitHub-Actions/Refusing-to-allow-an-integration-to-create-or-update/td-p/32472).

Can you elaborate more on this ?
> I manually created the *.yml file, so at least the publishing is working now.

**Comment by NickeManarin** [comment]
> did you just installed the plugin and pushed to master ? If you did something else, can you please share the steps to reproduce this issue?

First I tried adding this plugin using `vue add vue-cli-plugin-gh-pages-auto-deploy`. 

I forgot that I had uncommitted changes, so I select N to cancel adding this plugin when the warning about uncommitted changes appeared.

After committing and syncing, I added successfully this plugin.

Then I just staged the files and tried to sync, without success.

Btw, I already had the gh-pages branch on my repository, and it had stuff in it.

> Can you elaborate more on this ?

Since I was just unable to send the *.yml file with the action to my repository, I simply copied its content and created the action manually, thus removing the need to send the file (I reverted the commit and unstaged it).

> Btw, a similar issue is referenced here.
Thanks, if I find the same issue again, I'll test this solution:

https://github.com/gitextensions/gitextensions/issues/4916#issuecomment-557509451

I think that this issue can be closed, since it's external.
